### PR TITLE
DOPE-208 add an order by asc and desc + tests

### PR DESCRIPTION
**Pull request by martinagallati-ergon**
as per ticket DOPE-208, added an enum for ORDER BY types ASC and DESC, a function to add the clause to the query and tests

### CODE REVIEW

File: core/src/main/kotlin/ch/ergon/dope/resolvable/clause/select/factory/OrderBy.kt [hunk 1]
```diff
@@ -7,4 +7,9 @@ import ch.ergon.dope.validtype.StringType
 
 open class OrderBy(clauses: List<Clause>) : Limit(clauses) {
     fun orderBy(stringField: Field<StringType>): Limit = ClauseBuilder(clauses).orderBy(stringField)
+    fun orderBy(stringField: Field<StringType>, orderByType: OrderByType): Limit = ClauseBuilder(clauses).orderBy(stringField, orderByType)
+}
+
+enum class OrderByType(val type: String) {
+    ASC("ASC"), DESC("DESC")
```
Comment (jansigi) [inline core/src/main/kotlin/ch/ergon/dope/resolvable/clause/select/factory/OrderBy.kt:14]: Move DESC on new line

File: core/src/test/kotlin/ch/ergon/dope/OrderByTest.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package ch.ergon.dope
+
+import ch.ergon.dope.resolvable.clause.select.factory.OrderByType
+import junit.framework.TestCase.assertEquals
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+
+class OrderByTest {
+    private lateinit var builder: StringBuilder
```
Comment (jansigi) [inline core/src/test/kotlin/ch/ergon/dope/OrderByTest.kt:9]: Is this used?

**Review by jansigi — CHANGES_REQUESTED** [review]
Check Linter via: ./gradlew check

File: core/src/test/kotlin/ch/ergon/dope/OrderByTest.kt (hunk 2 above)
Comment (martinagallati-ergon) [reply core/src/test/kotlin/ch/ergon/dope/OrderByTest.kt:9]: no, removed it

**Review by sigristjan — APPROVED** [review]

**Review by jansigi — APPROVED** [review]

**Pull request merged**